examples/files.js

```diff
@@ -124,6 +124,7 @@ var files = {
 		"webgl_loader_utf8",
 		"webgl_loader_vrml",
 		"webgl_loader_vtk",
+		"webgl_loader_ww_meshspray",
```

Comment:
Maybe this one could be renamed to `webgl_loaader_obj2_workers`?

Reply:
What about `webgl_loader_worker_support`. The example is completely independent of `OBJLoader2` and it outlines the usage of `WorkerSupport`. I have to remove the import statement for OBJLoader2 in the example, btw.

Reply:
or `webgl_loader_worker_parser_simple`

Reply:
I renamed it to `webgl_loader_obj2_meshspray`.